WebAPI/Controllers/CategoriesController.cs

```diff
@@ -16,10 +16,27 @@ public CategoriesController(ICategoryService categoryService)
     }
 
     [HttpGet]
-    public async Task<ActionResult<List<CategoryDetailDto>>> GetAll()
+    public async Task<ActionResult<List<CategoryDetailDto>>> GetAll() =>
+                await _categoryService.GetAllAsync();
+
+    [HttpGet("{id}:guid")]
+    public async Task<ActionResult<string>> GetImage(Guid id)
+    {
+        var result = await _categoryService.GetByIdAsync(id);
+
+        return result.ImagePath;
+    }
+
+    [HttpPut]
+    public async Task<ActionResult<CategoryDetailDto>> UpdateImage(CategoryDetailDto updatedCategory)
     {
-        var result = await _categoryService.GetAllAsync();
+        var result = await _categoryService.UpdateAsync(updatedCategory);
+
+        if(result != null)
+        {
+            return result;
+        }
 
-        return Ok(result);
+        return BadRequest(result);
```

Comment:
You can simplify it like:
`return result ?? BadRequest(result)`

Reply:
But it gives an error. 
`Operator "??" cannot be applied to operands of type "ProductDetailDto" and "BadRequestObjectResult"`

Reply:
Ah, got it. My bad. Let's leave it as it is now, just add empty space between if, like `if (result != null)`